TeamCode/src/main/java/org/firstinspires/ftc/teamcode/hardware/RobotHardware.java

```diff
@@ -18,13 +16,14 @@ public static void setHardwareMap(HardwareMap map){
         hardwareMap = map;
     }
     public static void initAll() {
-        imu = hardwareMap.get(IMU.class, "imu");
-        motorFrontRight = hardwareMap.get(DcMotor.class, "FRmotor");
-        motorbackRight = hardwareMap.get(DcMotor.class, "BRmotor");
-        motorfrontLeft = hardwareMap.get(DcMotor.class, "FLmotor");
-        motorbackLeft = hardwareMap.get(DcMotor.class, "BLmotor");
+        DTmotorLeft = hardwareMap.get(DcMotor.class, "DTmotorLeft");
```

Comment:
DT is not following the java pattern camel case

Reply:
also, it doesn't seem a good practice to have all motors mixed up together if we're working with subsystem separately. it makes more sense to have access of the hardware map on the subsystem to get the motor

Reply:
still, if you want to keep all the hardware mapping in one file, I would suggest to separate by classes at least. but please, do not add subsystem logics on the hardware mapping file